Library/Homebrew/test/os/mac/pkgconfig_spec.rb

```diff
@@ -15,7 +15,11 @@
 # For indeterminable cases, consult https://opensource.apple.com for the version used.
 describe "pkg-config" do
   def pc_version(library)
-    path = HOMEBREW_LIBRARY_PATH/"os/mac/pkgconfig/#{MacOS.sdk_version}/#{library}.pc"
+    path = if MacOS.version >= :big_sur
+      HOMEBREW_LIBRARY_PATH/"os/mac/pkgconfig/#{MacOS.version}/#{library}.pc"
+    else
+      HOMEBREW_LIBRARY_PATH/"os/mac/pkgconfig/#{MacOS.sdk_version}/#{library}.pc"
+    end
```

Comment:
Sounds like this can just be `MacOS.version` in all cases?

There's some paths in `extend/os/mac/extend/ENV/` that need updating too.

Reply:
I thought that but I wasn't sure why it was using `sdk_version` in the first place at all, so I thought I'd play it safe. I'll change it then have a look in `extend/os/mac/extend/ENV/`.

Reply:
It was changed when the behaviour of `MacOS.version` changed, otherwise the tests would have broken.

It was originally `MacOS.version` and it should match what the pkg-config formula does.

Reply:
> `MacOS.version` in all cases

This would be preferable if it's possible and tests pass 👍🏻

Reply:
- test updated per @Bo98's feedback
- paths in `extend/os/mac/extend/ENV/` fixed
- verified that no other path definitions need fixing with `rg 'pkgconfig' $(brew --repo)`
- symlinks in `os/mac/pkgconfig` added

Let me know if there's anything else.

Reply:
I added `sdk_version` specifically to handle the Big Sur versioning. If it's not needed for `pkg-config` I think it can be removed completely.

Reply:
Do you mean the method `sdk_version`? If so, then it's still in use over here:

https://github.com/Homebrew/brew/blob/5521c76e663977f742d09fa03ca0f76fa250b15e/Library/Homebrew/os/mac/sdk.rb#L45-L48

I think removing it properly requires rethinking a lot of the logic of `os/mac/sdk.rb`.

Reply:
No, I meant the `sdk_version` method. Nevermind if it's also used in other places.

Reply:
Yes, I was referring to the `sdk_version` method. It's still in use, but I think it's not quite doing what was originally intended, so some refactoring may be in order.